README.md

```diff
@@ -118,8 +128,21 @@ Further, if there are any formula fields that are a third degree referential for
 * `key` (required): The key column within `fivetran_formula` you are querying. This argument is typically `field`.
 * `value` (required): The value column within `fivetran_formula` you are querying. This argument is typically `sql`.
 * `join_to_table` (required): The table with which you are joining the formula fields.
+* `added_inclusion_fields` (optional, default is none): The list of fields you want to be included in the macro. If no fields are selected then all fields will be included.
+* `not_null_value` (optional, default is true): Used by the macro to identify if the `null` fields within the `view_sql` column should be included.
 ----
 
+### sfdc_star_exact_ ([source](macros/sfdc_star_exact.sql))
+This macro mirrors the [dbt_utils.star()](https://github.com/dbt-labs/dbt-utils#star-source) macro with the minor adjustment to properly work with the return results of the [dbt_utils.get_column_values()](https://github.com/dbt-labs/dbt-utils#get_column_values-source). The major change being how the return result of the dbt_utils.get_column_values() macro returns results with single quotes, while the dbt_utils.star() macro exclusively requires double quotes.
```

Comment:
this is probably too late lol but did you try using the `replace` jinja filter to swap single quotes with double ones and then provide that to `dbt_utils.star()`? just so we don't have to worry about maintaining this ourselves 🤓

Reply:
Funny enough I did. It didn't work strangely? This one was confusing me a lot and this was the only solution I could get to provide the same output. 

It may be worth a shot to try again since I remember being deep into the weeds here and taking another fresh look may be a good option!

Reply:
I was able to go back in and try the replace filter; however, it was not working ☹️ 

The replace filter did correctly update the `'` to `"`, but it still excluded sub string fields. It wasn't until I used the macro in this package where the substring fields were not excluded. This may be something we want to add to fivetran_utils or open a PR in dbt_utils to allow others to use.

Reply:
gotcha -- might wanna move this to fivetran_utils 🤷

Reply:
Yeah I think that may be the call. In an effort of time I will keep this in here with the intention of moving to fivetran_utils in the future.